hacker-news-app/src/App.css

```diff
@@ -0,0 +1,24 @@
+@import './Components/LogInPage/loginStyles.css';
+
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #09d3ac;
+}
```

Comment:
I doubt that these styles are being used anywhere, right? If that is the case - lets cleanup 😉

Reply:
This is very true! I will clean it up now.